depthai_nodes/ml/parsers/yolo.py

```diff
@@ -146,7 +187,9 @@ def run(self):
                 for o in outputs_names
             ]
 
-            if any("kpt_output" in name for name in layer_names):
+            if self.bbox_only:
```

Comment:
Can we set self._DET_MODE=True at the end if it is neither kpts or seg mode (`else` at the end). Because it has to be one of those right? And then we can remove self.bbox_only attribute.

Reply:
There might be a problem with this approach. For example the `YoloP` model has both bboxes and masks, so in that case if we don't explicitly define the `bbox_only` flag it will catch the `_masks` in the layer names and the `_SEG_MODE` will be activate. Again this flag was needed (IMO) for dealing with the special case of `YoloP`.

Reply:
This is where ParserV2 comes into play because only layers regarding the detection will be passed into this parser where as layers that should use segmentation parser will not be looked at here.